libs/common-services/src/main/java/uk/gov/di/ipv/core/library/tracing/TracingHttpClient.java

```diff
@@ -33,18 +34,25 @@ public class TracingHttpClient extends HttpClient {
     private static final Logger LOGGER = LogManager.getLogger();
     private static final List<String> HTTP_ERROR_MESSAGES_TO_RETRY =
             List.of("GOAWAY received", "Connection reset");
+    private static final Duration MAX_CLIENT_AGE = Duration.ofHours(1);
     private final HttpClient baseClient;
     private final AWSXRayRecorder recorder;
+    private final Instant creationTime;
 
     private TracingHttpClient(HttpClient baseClient) {
         this.baseClient = baseClient;
         this.recorder = AWSXRay.getGlobalRecorder();
+        this.creationTime = Instant.now();
     }
 
     public static HttpClient newHttpClient() {
         return new TracingHttpClient(HttpClient.newHttpClient());
     }
 
+    private boolean isClientTooOld() {
```

Comment:
I think this needs an explanatory comment - e.g. linking to the Jira ticket, otherwise it's a bit unclear what's going on.